scripts/setup-windows.ps1

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
```

Comment:
Any tools needed to build should go in here if possible. CMake was the only one I could think of at the time.